requirements/base.txt

```diff
@@ -1,8 +1,10 @@
 celery
 django>=1.11,<2.2
 django_compressor
+pillow
```

Comment:
why is this needed?

Reply:
then it can go in requirements/tests.txt ?

Reply:
I moved it tho any project that needs to use images will end up installing it